app/src/main/res/layout/fragment_arinstructions.xml

```diff
@@ -51,7 +51,7 @@
             android:id="@+id/next_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="@string/next_step" />
```

Comment:
Yes, jag var feg. Fatta inte vad skillnande var först. sen på peka alex. I'm a :cow: + :a: + r + :dart:  - :art: and a :chicken: